active/NumberBox/NumberBox.md

```diff
@@ -0,0 +1,220 @@
+```
+
+```XAML
+<TextBlock AutomationProperties.Name="NumberBox for custom input" 
+    Header="Enter equation:" 
+    PlaceholderText="00 ft 00 in"
+    CustomFormatString="## ft ## in"
+    StripLeadingZeros="False" />
+```
+
+### Add increment and decrement stepping
+
+XAML
+```XAML
+<TextBlock AutomationProperties.Name="NumberBox for items in order" 
+    Header="Order:" 
+    Text="2"
+    StepSize="2"
+    UpDownArrowsEnabled="True"
+    UpDownDragEnabled="True" />
+```
+
+### Add a prefix and suffix 
+
+XAML
+```XAML
+<TextBlock AutomationProperties.Name="NumberBox for IP address" 
+    Header="Enter equation:" 
+    PlaceholderText="000.000.000.000" 
+    Prefix="IP Address:" />
+```
+
+XAML
+```XAML
+<TextBlock AutomationProperties.Name="NumberBox for cents" 
+    Header="Enter equation:" 
+    PlaceholderText="0.00" 
+    Suffix="cents" />
+```
+
+### Add mininum and maximum values
+
+XAML
+```XAML
+<TextBlock AutomationProperties.Name="NumberBox for items in order" 
+    Header="Order:" 
+    PlaceholderText="1" 
+    MinValue="1"
+    MaxValue="99" />
+```
+
+## Remarks
+<!-- Explanation and guidance that doesn't fit into the Examples section. -->
+
+<!-- APIs should only throw exceptions in exceptional conditions; basically,
+only when there's a bug in the caller, such as argument exception.  But if for some
+reason it's necessary for a caller to catch an exception from an API, call that
+out with an explanation either here or in the Examples -->
+
+## API Notes
+<!-- Option 1: Give a one or two line description of each API (type
+and member), or at least the ones that aren't obvious
+from their name.  These descriptions are what show up
+in IntelliSense. For properties, specify the default value of the property if it
+isn't the type's default (for example an int-typed property that doesn't default to zero.) -->
+
+<!-- Option 2: Put these descriptions in the below API Details section,
+with a "///" comment above the member or type. -->
+
+## API Details
+<!-- The exact API, in MIDL3 format (https://docs.microsoft.com/en-us/uwp/midl-3/) -->
+
+```c++ 
+enum NumberBoxFormatType
+{
+    IPAddress,
+    InternationalTelephone,
+    Currency,
+};
+
+unsealed runtimeclass NumberBox : Windows.UI.Xaml.Controls.TextBox
+{
+    NumberBox();
+    
+    String Prefix;
+    String Suffix;
+    
+    Boolean AcceptsCalculation;
+    Boolean UpDownArrowsEnabled;
+    Boolean UpDownDragEnabled;
+  
+    NumberBoxFormatType FormatType;
+    String CustomFormatString;
+    Boolean StripLeadingZeros;
+   
+    Integer MinValue;
+    Integer MaxValue;
+
+    static Windows.UI.Xaml.DependencyProperty PrefixProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty SuffixProperty{ get; };
+    
+    static Windows.UI.Xaml.DependencyProperty AcceptsCalculationProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty UpDownArrowsEnabledProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty UpDownDragEnabledProperty{ get; };
```

Comment:
Will any UI affordance for dragging support be provided by default? If so what, and will it be customizable?

Reply:
That is prospective still. For this detail I will work with our designer and Fluent Design System representatives to pin down the user interaction model and whether dragging, scrolling, or arrow stepping will be supported by default or enableable. The most limiting factor in this respect will be the paradigm set by accessibility requirements and what is needed or perhaps prohibited among these options. Are any of these specifically a make or break need for you @mrlacey?

Reply:
Nothing is essential here. That what "UpDownDragEnabled" really means or how it will be surfaced aren't clear is my biggest concern here.

Reply:
As noted below,I have added the [API Note](https://github.com/microsoft/microsoft-ui-xaml-specs/commit/b7786665bb18f18d02a6133abedaa8c6d9300fbd) to align hyper drage with WinRT XAML Toolkit's NumericUpDown/Blender which, I believe, has already solved this portion of our control equation. Are there any gaps you foresee with adopting this work, @mrlacey?